torch/fx/experimental/proxy_tensor.py

```diff
@@ -1245,6 +1255,11 @@ def __torch_function__(
         return func(*args, **kwargs)
 
 
+_temp_remove_pre_dispatch_torch_function_mode = _make_temp_remove_mode_context_manager(
+    PreDispatchTorchFunctionMode
+)
```

Comment:
@mlazos do you have a test case that demonstrates what's going on?

Reply:
Basically the reason I did this was there were some failures on the cond/while loop tests, I briefly discussed with @avikchaudhuri and we thought disabling this mode was fine since dynamo was previously not tracing it anyway, popping this preserves the existing behavior.

Reply:
I think we should address the test failures. What tests are failing? I can help to see what happened.

Reply:
Thanks Yidi, so the test failures are `test_cond_with_module_stack_export_with` on the next PR in this stack. Basically we graph break at line 1221 of proxy_tensor.py. If we're tracing is it strictly necessary that we keep track of those counts?